### PR TITLE
Added Custom Assignment Cascade feature documentation.

### PR DESCRIPTION
**Pull request by MaceWindu1**
title: TALXIS Custom Assignment Cascade
tagline: Enables advanced assignment cascade configurations

### CODE REVIEW

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md
Comment (JanKrupicka) [inline src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:23]: I would add "Autofills on save" or something like that so they know they don't have to do anything with this

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md
Comment (JanKrupicka) [inline src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:30]: I might also write that inactive records means the logic does not trigger

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md
Comment (JanKrupicka) [inline src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:38]: And maybe add a sample of execution

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md
Comment (MaceWindu1) [reply src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:23]: I think it's obvious read-only fields should be left untouched.

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md
Comment (MaceWindu1) [reply src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:30]: Good point.

File: src/en/developer-guide/applications/utilities/custom-assignment-cascade.md
Comment (JanKrupicka) [reply src/en/developer-guide/applications/utilities/custom-assignment-cascade.md:23]: Sure Im ok with that.

**Comment by MaceWindu1** [comment]
Waiting for review from @honzakostejn.

**Review by honzakostejn — APPROVED** [review]

**Pull request merged**
